gym-unity/gym_unity/envs/__init__.py

```diff
@@ -24,42 +24,29 @@ class UnityGymException(error.Error):
 GymStepResult = Tuple[np.ndarray, float, bool, Dict]
 
 
-class UnityEnv(gym.Env):
+class UnityToGymWrapper(gym.Env):
     """
     Provides Gym wrapper for Unity Learning Environments.
     """
 
     def __init__(
         self,
-        environment_filename: str,
-        worker_id: int = 0,
+        unity_env: BaseEnv,
         use_visual: bool = False,
         uint8_visual: bool = False,
         flatten_branched: bool = False,
-        no_graphics: bool = False,
         allow_multiple_visual_obs: bool = False,
     ):
         """
         Environment initialization
-        :param environment_filename: The UnityEnvironment path or file to be wrapped in the gym.
-        :param worker_id: Worker number for environment.
+        :param unity_env: The Unity BaseEnv to be wrapped in the gym.
```

Comment:
Add a note that the environment will be closed when the UnityToGymWrapper is closed.